docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -194,6 +194,31 @@ Modify your `package.json` commands:
   }
 ```
 
+If you are using Next.js >= v12.1.1, you can make use of their [Modularize Imports](https://nextjs.org/docs/advanced-features/compiler#modularize-imports) feature.
+This does not require adding Babel configuration, so you can still use [SWC](https://nextjs.org/docs/advanced-features/compiler#why-swc) if you prefer.
+
+Create a `next.config.js` file in the root directory:
+
+```js
+/* next.config.js */
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  ...
+  experimental: {
```

Comment:
It doesn't seems to be under `experimental` in https://nextjs.org/docs/advanced-features/compiler#modularize-imports

Reply:
It's recently [promoted](https://github.com/vercel/next.js/pull/44240)